src/csharp/Microsoft.Spark.E2ETest/IpcTests/ML/Feature/BucketizerTests.cs

```diff
@@ -47,12 +49,20 @@ public void TestBucketizer()
             Assert.Equal(expectedInputCol, bucketizer.GetInputCol());
             Assert.Equal(expectedOutputCol, bucketizer.GetOutputCol());
             Assert.Equal(expectedSplits, bucketizer.GetSplits());
+            
+            using (var tempDirectory = new TemporaryDirectory())
+            {
+                var savePath = Path.Join(tempDirectory.Path, "bucket");
```

Comment:
Can you go through `var` usages (https://github.com/dotnet/spark/blob/master/docs/coding-guidelines/csharp-coding-style.md)? Thanks!